www/sesm00/sp/views/orderView.php

```diff
@@ -0,0 +1,145 @@
+<?php include __DIR__ . '/../components/header.php'; ?>
+<?php include __DIR__ . '/../components/navigation.php'; ?>
+<?php ?>
+<div class="min-vh-100 position-relative page-container">
+    <div class="container py-2">
+        <h1 class="mb-4">Objednávka</h1>
+        <?php include __DIR__ . '/../components/errorPrinter.php'; ?>
+        <form method="post">

```

Comment:
je mozne tento formular predem vyplnit pomoci udaju z registrace?

Reply:
Možné by to bylo, ale nezbyl mi čas to implementovat